test/jdk/java/lang/reflect/AccessibleObject/TrySetAccessibleTest.java

```diff
@@ -193,11 +194,24 @@ public void testJavaLangClass() throws Exception {
 
         // non-public constructor
         Constructor<?> ctor
-            = Class.class.getDeclaredConstructor(ClassLoader.class, Class.class);
+            = Class.class.getDeclaredConstructor(ClassLoader.class, Class.class, ProtectionDomain.class);
         AccessibleObject[] ctors = { ctor };
 
         assertFalse(ctor.trySetAccessible());
         assertFalse(ctor.canAccess(null));
     }
 
+    /**
+     * Test that some Class fields cannot be make accessible.
+     */
+    public void testJavaLangClassFields() throws Exception {
+
+        Field pd = null; // implied?
+        try {
+            // This field is explicitly hidden by reflection.
+            pd = Class.class.getDeclaredField("protectionDomain");
+        } catch (NoSuchFieldException expected) { }
+
+        assertTrue(pd == null);
+    }
```

Comment:
```suggestion
    public void testJavaLangClassFields() throws Exception {
        try {
            // This field is explicitly hidden from reflection.
            Class.class.getDeclaredField("protectionDomain");
            assertTrue(false);
        } catch (NoSuchFieldException expected) { }
    }
```
The above is more in-style with the other test cases.